content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -72,6 +72,17 @@ To setup unified service tagging in a containerized environment:
 
 3. Configure your environment that corresponds to your container orchestration service based on either full configuration or partial configuration as detailed below.
 
+#### Automatic version tagging for containerized environments
```

Comment:
Any reason to not include the information in the Kubernetes/Docker/ECS section and add that DD_ENV is optional in these env?